### PR TITLE
Add Mnist Demo

### PR DESCRIPTION
**Pull request by ChildishChange**

### CODE REVIEW

File: projects/MnistDemo/MnistModel/MNISTModelLibrary.csproj [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  <ItemGroup>
+    <None Include="packages.config">
+      <SubType>Designer</SubType>
+    </None>
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Import Project="..\packages\Microsoft.ML.Scoring.1.0.3\build\net45\Microsoft.ML.Scoring.targets" Condition="Exists('..\packages\Microsoft.ML.Scoring.1.0.3\build\net45\Microsoft.ML.Scoring.targets')" />
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
+    <PropertyGroup>
+      <ErrorText>这台计算机上缺少此项目引用的 NuGet 程序包。使用“NuGet 程序包还原”可下载这些程序包。有关更多信息，请参见 http://go.microsoft.com/fwlink/?LinkID=322105。缺少的文件是 {0}。</ErrorText>
```
Comment (shishaochen) [inline projects/MnistDemo/MnistModel/MNISTModelLibrary.csproj]: Please set VS region to en-US.

**Comment by shishaochen** [comment]
Could you add localization to this .NET program?

**Comment by ChildishChange** [comment]
@shishaochen Hi, thanks for review. The current default language of this program is English.

File: projects/MnistDemo/MNIST.App/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// 有关程序集的一般信息由以下
```
Comment (shishaochen) [inline projects/MnistDemo/MNIST.App/Properties/AssemblyInfo.cs]: Please switch the region of VS to en-US. Thanks.

File: projects/MnistDemo/MNISTModelLibrary/MNISTModelLibrary.csproj [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  <ItemGroup>
+    <None Include="packages.config">
+      <SubType>Designer</SubType>
+    </None>
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Import Project="..\packages\Microsoft.ML.Scoring.1.0.3\build\net45\Microsoft.ML.Scoring.targets" Condition="Exists('..\packages\Microsoft.ML.Scoring.1.0.3\build\net45\Microsoft.ML.Scoring.targets')" />
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
+    <PropertyGroup>
+      <ErrorText>这台计算机上缺少此项目引用的 NuGet 程序包。使用“NuGet 程序包还原”可下载这些程序包。有关更多信息，请参见 http://go.microsoft.com/fwlink/?LinkID=322105。缺少的文件是 {0}。</ErrorText>
```
Comment (shishaochen) [inline projects/MnistDemo/MNISTModelLibrary/MNISTModelLibrary.csproj]: Please switch the region of VS to en-US. Thanks.

File: projects/MnistDemo/MNIST.App/MNIST.App.csproj [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    <None Include="App.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\MnistModel\MNISTModelLibrary.csproj">
+      <Project>{e156950d-6116-4bef-8279-51bee6765b3b}</Project>
+      <Name>MNISTModelLibrary</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+</Project>
```
Comment (shishaochen) [inline projects/MnistDemo/MNIST.App/MNIST.App.csproj]: Please add EOL at the end of files. So do other files.

File: projects/MnistDemo/MNISTModelLibrary/Properties/AssemblyInfo.cs [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("MNISTModelLibrary")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyProduct("MNISTModelLibrary")]
+[assembly: AssemblyCopyright("Copyright © OpenMindStudio, Microsoft 2018")]
```
Comment (shishaochen) [inline projects/MnistDemo/MNISTModelLibrary/Properties/AssemblyInfo.cs]: Please use "Microsoft" only.

**Pull request merged**